tracbev/__main__.py

```diff
@@ -1,112 +1,134 @@
-import pandas as pd
-import geopandas as gpd
-import numpy as np
-import configparser as cp
 import argparse
-from datetime import datetime
+import configparser as cp
 import pathlib
 
+from datetime import datetime
+
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+
 import tracbev.usecase as uc
 
 
 def parse_data(args):
     # read config file
     parser = cp.ConfigParser()
-    scenario_path = pathlib.Path('scenarios', args.scenario)
-    cfg_file = pathlib.Path(scenario_path, 'tracbev_config.cfg')
-    data_dir = pathlib.Path('data')
+    scenario_path = pathlib.Path(".").resolve().parent / "scenarios" / args.scenario
```

Comment:
```suggestion
    scenario_path = pathlib.Path(".").resolve() / "scenarios" / args.scenario
```

This way it works for me. Otherwise i get the error from my comment